sympy/physics/control/lti.py

```diff
@@ -774,6 +804,18 @@ def var(self):
         """
         return self._var
 
+    @property
+    def num_inputs(self):
+        return self._num_inputs
+
+    @property
+    def num_outputs(self):
+        return self._num_outputs
+
+    @property
+    def shape(self):
+        return self._shape
```

Comment:
Why not just `return self._num_inputs, self._num_outputs`? Are these ever different?